### PR TITLE
Qpcr: tags clashes

### PR DESCRIPTION
**Pull request by Proskurina**

### CODE REVIEW

**Comment by Proskurina** [comment]
Now, I am not sure it should be a button, or maybe it should always give more detailed tags error message? It does not seem to be slow enough to move it to separate button.

File: lib/sample_manifest_excel/sample_manifest_excel/tags/clashes_finder.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module SampleManifestExcel
+  module Tags
+    module ClashesFinder
+      # This method takes an array of tags combinations, i.e.
+      # [['AA', 'TT'], ['CC', 'GG'], ['ACG', 'CGT'], ['AA', 'TT'], ['A', 'TCG'], ['CC', 'GG']]
+      # and returns a hash, indicating the indexes of tags clahses, in this example
+      # {['AA', 'TT'] => [0, 3], ['CC', 'GG'] => [1, 5]}
+
```
Comment (Proskurina) [inline lib/sample_manifest_excel/sample_manifest_excel/tags/clashes_finder.rb]: combinations_with_indexes

File: spec/lib/sample_manifest_excel/tags_spec.rb [hunk 2]
```diff
@@ -55,4 +55,22 @@ def initialize(upload)
...
+      result = tag_clashes_finder.find_tags_clash(tags_oligos_combinations)
+      expect(result).to eq(['AA', 'TT'] => [0, 3])
+      message = tag_clashes_finder.create_tags_clashes_message(result)
+      expect(message).to eq('Same tags AA, TT are used on rows 1, 4.')
+      first_row = 5
+      message_for_manifest = tag_clashes_finder.create_tags_clashes_message(result, first_row)
+      expect(message_for_manifest).to eq('Same tags AA, TT are used on rows 6, 9.')
+    end
+  end
 end
```
Comment (Proskurina) [inline spec/lib/sample_manifest_excel/tags_spec.rb]: add tests for no clashes/ empty array
